lib/tapioca/helpers/rbi_helper.rb

```diff
@@ -107,7 +107,7 @@ def as_non_nilable_type(type)
 
     sig { params(name: String).returns(T::Boolean) }
     def valid_method_name?(name)
-      Prism.parse("def self.#{name}(a); end").success?
+      Prism.parse_success?("def self.#{name}(a); end")
```

Comment:
This should be done for line 115 as well: https://github.com/Shopify/tapioca/pull/2067/files#diff-d83dd54865c126791df7c2e1c33243083422b0e90f937a4bdab3be90dca5f257R115